server/datastore/mysql/migrations/tables/20230411102858_CreateHostBootstrapPackagesTable.go

```diff
@@ -0,0 +1,31 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230411102858, Down_20230411102858)
+}
+
+func Up_20230411102858(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+          CREATE TABLE host_mdm_apple_bootstrap_packages (
+	    host_uuid    varchar(127) NOT NULL,
+            command_uuid varchar(127) NOT NULL,
+
+            PRIMARY KEY (host_uuid, command_uuid),
+            FOREIGN KEY (command_uuid) REFERENCES nano_commands (command_uuid) ON DELETE CASCADE
+          )`)
```

Comment:
@mna @gillespi314 I'm not sure if this table is the right approach, do we want a more general "fleet commands" table? this table would still need a column with a flag/enum we can use to distinguish which commands are related to bootstrap packages.

up to you!

Reply:
Hmm yeah good question, this is the only command that we need this for at the moment, right? The disk encryption profile is also treated a bit as a special case, but it does not need something like this AFAIR? Given that this is the first need for this, it could make sense to use it like you proposed, and rename it to something more general if and when we need something similar for other "special commands"?